### PR TITLE
Add release note for network port devices' default configuration profiles

### PR DESCRIPTION
**Pull request by rvykydal**
Relase note for https://github.com/rhinstaller/anaconda/pull/5703

### CODE REVIEW

File: docs/release-notes/no-network-port-device-default-profiles.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:Type: Network
+:Summmary: Do not create default network profiles for network port devices
+
+:Description:
+    Traditionally Anaconda creates default persistent network profiles (ifcfg files or keyfiles) for every supported wired network device.  We would like to move towards creating profiles only for devices explicitly configured by installer. As a step in this direction do not create such files for devices used as ports of a virtual device (eg bond device) configured by installer, unless they were explicitly configured separately (for example in early stage from boot options).
```
Comment (KKoukiou) [inline docs/release-notes/no-network-port-device-default-profiles.rst:5]: s/eg bond device/e.g., bond device

**Review by KKoukiou — COMMENTED** [review]
The note is perfect, only tiny typo in the commit message.

s/confiuration/configuration

File: docs/release-notes/no-network-port-device-default-profiles.rst (hunk 1 above)
Comment (rvykydal) [reply docs/release-notes/no-network-port-device-default-profiles.rst]: Updated.

**Comment by KKoukiou** [comment]
> The note is perfect, only tiny typo in the commit message.
> 
> s/confiuration/configuration

@rvykydal I think you missed this one :) Will update it quickly for you and merge.

**Comment by rvykydal** [comment]
> > The note is perfect, only tiny typo in the commit message.
> > s/confiuration/configuration
> 
> @rvykydal I think you missed this one :) Will update it quickly for you and merge.

Thank you :)

**Pull request merged**